Change e. g. to e.g. (#13083)
JabRef/jabref: jablib/src/main/resources/resource/layout/din1505/README.txt

This is a custom export to allow a reference output in DIN1505 conform style. That means e.g. authors in capitals and Book/article title in italic.

> This export is not yet complete and some issues cannot be resolved without extension of the internal format modifier. E.g., the last author is separates by '&' but should have a ',' only.

Reviewer: ```suggestion
This export is not yet complete and some issues cannot be resolved without extension of the internal format modifier. E.g., the last author is separated by '&' which should be a ','.
```
